pwndbg/chain.py

```diff
@@ -181,3 +186,37 @@ def format(
         return enhanced
 
     return arrow_right.join(rest) + arrow_left + enhanced
+
+def _format_flags_value(flags: List[Flag], value: int):
```

Comment:
Ideally, we should add some tests for this function

Reply:
Done